quickstart-templates/workload-identity-creates-workload-identities/README.md

```diff
@@ -0,0 +1,70 @@
+# Create Workload Identity Federation that can create other workload identity federations
+
+## The Idea
+
+Create federated identity credentials (FIC) at appropriate scopes and controls to deploy company and project specific resource. The process is rooted in a Git pull request review process in order to introduce checkpoints, audit trail and repeatability to process. This should restrict the need for manual work requiring high privileges.
+
+## Example scenario
+
+A high-privilege user creates an initial, bootstrap workload identity federation token between Azure and a certain GitHub repository. The workload in this repository is then allowed to access Microsoft Entra to create further workload identity federation tokens and assign them to other repositories.
+
+The goal is to create a GitHub repository that is privileged enough to create further, more specific federated identifier according to needs of specific projects. This way creating resources can be support by pull request reviews. Together with application roles and restricting the federated identity credential (FIC) access to a certain repository, one can reduce the "blast radius" for any given project and repository and human operators.
+
+## Benefits
+
+- **Enhanced Security:** Restricting high-privilege usage reduces the attack surface and potential for unauthorized access.
+- **Scalability:** Facilitates the growth of projects by enabling the creation of new identity federations as needed.
+- **Efficient Management:** Streamlines the process of managing resource access across multiple projects and teams.
+
+## Usage could look something like this
+
+### az cli
+
+```sh
+# Sign in to Azure.
+az login
+
+# Sign in to GitHub.
+gh auth login
+
+# Deploy the Bicep template and capture the output as JSON.
+deploymentOutput=$(az deployment tenant create --name bootstrap --location WestEurope --template-file ./main.bicep --parameters ./main.bicepparam --output json)
```

Comment:
Would it be more scoped to create the creator app scoped to a resource group (this is more about scoping the Azure access - Entra/Graph don't know anything about Azure scoping like resource groups or subscriptions)?
Or do we think each developer that is creating their project workload identities will do so under their own resource group scope?